tests/acceptance/features/apiAntivirus/antivirus.feature

```diff
@@ -127,3 +127,73 @@ Feature: antivirus
       | old              |
       | new              |
       | spaces           |
+
+
+  Scenario Outline: upload a file with virus to a user share
+    Given using <dav-path-version> DAV path
+    And user "Brian" has been created with default attributes and without skeleton files
+    And user "Alice" has created folder "uploadFolder"
+    And user "Alice" has shared folder "uploadFolder" with user "Brian" with permissions "all"
+    And user "Brian" has accepted share "/uploadFolder" offered by user "Alice"
+    When user "Brian" uploads file "filesForUpload/filesWithVirus/eicar.com" to "/Shares/uploadFolder/file.txt" using the WebDAV API
+    Then the HTTP status code should be "201"
+    And user "Brian" should get a notification with subject "Virus found" and message:
+      | message                                                                   |
+      | Virus found in file.txt. Upload not possible. Virus: Win.Test.EICAR_HDB-1 |
+    And as "Brian" file "/Shares/uploadFolder/file.txt" should not exist
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+
+
+  Scenario: upload a file with virus to a user share using DAV path spaces
+    Given using spaces DAV path
+    And user "Brian" has been created with default attributes and without skeleton files
+    And user "Alice" has created folder "uploadFolder"
+    And user "Alice" has shared folder "uploadFolder" with user "Brian" with permissions "all"
+    And user "Brian" has accepted share "/uploadFolder" offered by user "Alice"
+    When user "Brian" uploads a file inside space "Shares" with content "X5O!P%@AP[4\PZX54(P^)7CC)7}$EICAR-STANDARD-ANTIVIRUS-TEST-FILE!$H+H*" to "/uploadFolder/file.txt" using the WebDAV API
+    Then the HTTP status code should be "201"
+    And user "Brian" should get a notification with subject "Virus found" and message:
+      | message                                                                   |
+      | Virus found in file.txt. Upload not possible. Virus: Win.Test.EICAR_HDB-1 |
+    And as "Brian" file "/uploadFolder/file.txt" should not exist
+
+
+  Scenario Outline: upload a file with virus to a group share
```

Comment:
```suggestion
  Scenario Outline: upload a virus file to a group share
```

Reply:
I think "file with virus" and "virus file" same